tests/bearlib/aspects/ClassTest.py

```diff
@@ -1,12 +1,14 @@
 from coalib.bearlib.aspects import aspectclass
 
-import pytest
+import unittest
 
 
 class AspectClassTest:
 
     def test_subaspect_without_definition(self, RootAspect):
-        with pytest.raises(TypeError):
+        testcase = unittest.TestCase()
+        error_str = 'must be an instance of one of'
+        with testcase.assertRaisesRegex(TypeError, error_str):
```

Comment:
Just `with self.assertRaises(TypeError):` -- the other two lines are unnecessary.